CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+28 April 2016 - 0.6.3
```

Comment:
please fix the changelog.

put them all under one heading, and mark it 'x.x.x' for us to change later.